Baby Monitor/Resources/Constants.swift

```diff
@@ -16,6 +16,9 @@ enum Constants {
 
     /// A limit per which one notification can be sent.
     static let notificationRequestTimeLimit: TimeInterval = 60
+
+    /// A time after which the video stream should be hidden.
+    static let videoStreamingHiddenAfterTime: TimeInterval = 60
```

Comment:
You could consider something like `videoStreamVisibilityTimeLimit` to keep it in similar pattern as the variable above - but it's entirely optional

Reply:
Very nice note :)